tests/grammar/vyper.lark

```diff
@@ -0,0 +1,148 @@
+// Vyper grammar for Lark
+// Adapter from Lark repo. https://github.com/lark-parser/lark/blob/master/examples/python3.lark
+// Adapted from: https://docs.python.org/3/reference/grammar.html
+// Adapted by: Erez Shinan
+
+// Start symbols for the grammar:
+//       single_input is a single interactive statement;
+//       file_input is a module or sequence of commands read from an input file;
+//       eval_input is the input for the eval() functions.
+// NB: compound_stmt in single_input is followed by extra NEWLINE!
+single_input: _NEWLINE | simple_stmt | compound_stmt _NEWLINE
+file_input: (_NEWLINE | stmt)*
```

Comment:
This should be the only type we need, right?

Reply:
Correct